src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -119,119 +130,16 @@ public void addLoginListener(LoginListener listener) {
-   * this transport will throw an exception, for example,
-   * "com.google.api.client.http.HttpResponseException: 401 Unauthorized"
-   *
-   * @throws IllegalStateException if no user is currently signed in
-   */
-  public HttpRequestFactory createRequestFactory() {
-    Preconditions.checkState(isLoggedIn);
-
-    return transport.createRequestFactory(oAuth2Credential);
-  }
-
-  /**
-   * Makes a request to get an OAuth2 access token from the OAuth2 refresh token
-   * if it is expired.
-   *
-   * @return an OAuth2 token
-   * @throws IllegalStateException if no user is currently signed in
-   * @throws IOException if something goes wrong while fetching the token
-   */
-  public String fetchAccessToken() throws IOException {
-    Preconditions.checkState(isLoggedIn);
-
-    if (accessTokenExpiryTime == 0) {
-      return fetchOAuth2Token();
-    }
-
-    long currentTime = System.currentTimeMillis() / 1000;
-    if (currentTime >= accessTokenExpiryTime) {
-      return fetchOAuth2Token();
-    }
-    return accessToken;
-  }
-
-  public String fetchOAuth2ClientId() {
-    return clientId;
-  }
-
-  public String fetchOAuth2ClientSecret() {
-    return clientSecret;
-  }
-
-  /**
-   * Returns the OAuth2 refresh token.
-   *
-   * @throws IllegalStateException if no user is currently signed in
-   */
-  public String fetchOAuth2RefreshToken() {
-    Preconditions.checkState(isLoggedIn);
-
-    return refreshToken;
-  }
-
-  /**
-   * Makes a request to get an OAuth2 access token from the OAuth2 refresh
-   * token. This token is short lived.
-   *
-   * @return an OAuth2 token
-   * @throws IllegalStateException if no user is currently signed in
-   * @throws IOException if something goes wrong while fetching the token
-   *
-   */
-  public String fetchOAuth2Token() throws IOException {
-    Preconditions.checkState(isLoggedIn);
-
-    try {
-      GoogleRefreshTokenRequest request = new GoogleRefreshTokenRequest(
-          transport, jsonFactory, refreshToken, clientId, clientSecret);
-      GoogleTokenResponse authResponse = request.execute();
-      accessToken = authResponse.getAccessToken();
-      oAuth2Credential.setAccessToken(accessToken);
-      accessTokenExpiryTime = System.currentTimeMillis() / 1000
-          + authResponse.getExpiresInSeconds();
-    } catch (IOException ex) {
-      loggerFacade.logError("Could not obtain an OAuth2 access token.", ex);
-      throw ex;
-    }
-    persistCredentials();
-    return accessToken;
-  }
-
-  public Credential getCredential() {
-    if (oAuth2Credential == null) {
-      oAuth2Credential = makeCredential();
-    }
-    return oAuth2Credential;
-  }
-
-  /**
-   * @return the user's email address, or the empty string if the user is logged
-   *         out, or null if the user's email couldn't be retrieved
-   */
-  public String getEmail() {
-    return email;
-  }
-
-  /**
    * @return true if the user is logged in, false otherwise
    */
   public boolean isLoggedIn() {
```

Comment:
Do you think you should flag this as `@deprecated`?  Or revise the javadocs — there is at least one account.